src/components/OrderList/index.tsx

```diff
@@ -11,12 +11,9 @@ import OrderListItem from './OrderListItem';
 import Orders from './Orders';
 
 export const OrderCellWidth = {
-  Symbol: 70,
-  Id: 300,
-  Side: 50,
-  Filled: 100,
-  CreatedDate: 150,
-  Actions: 60
+  Symbol: 100,
```

Comment:
is it your intention to remove "widths" for other columns?

Reply:
Yes, I did it to make `Orders` and `Trades` tabs look similar.